lib/Drush/Commands/core/CliCommands.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drush\Commands\core\CliCommands.
```

Comment:
Yeah, this is just automated from my IDE (from the days we needed that). I'll remove!